GooglePay/src/main/java/com/braintreepayments/api/GooglePayCardNonce.java

```diff
@@ -214,6 +220,15 @@ public BinData getBinData() {
         return binData;
     }
 
+    /**
+     * @return The card network. This card network value should not be displayed.
```

Comment:
Take it or leave it comment - 
```suggestion
     * @return The card network. This card network value should not be displayed to the buyer.
```